src/main/java/uk/gov/companieshouse/overseasentitiesapi/validation/AddressDtoValidator.java

```diff
@@ -0,0 +1,69 @@
+package uk.gov.companieshouse.overseasentitiesapi.validation;
+
+import org.springframework.stereotype.Component;
+import uk.gov.companieshouse.overseasentitiesapi.model.dto.AddressDto;
+import uk.gov.companieshouse.overseasentitiesapi.validation.utils.StringValidators;
+import uk.gov.companieshouse.service.rest.err.Errors;
+
+@Component
+public class AddressDtoValidator {
+
+    public Errors validate(String field, AddressDto addressDto, Errors errs, String loggingContext) {
```

Comment:
Would suggest to not bother shortening 'errs' as a variable name, Just call it 'errors' - not that long and improves code readability.

What exactly is the 'field' parameter here? Is it a 'field name' or a 'field value'. What I'm getting at is, can it be renamed to better describe what (string) content it contains?

Reply:
Updated the name of the method.